DATA605/projects/DATA605_Projects.md

```diff
@@ -1,202 +1,729 @@
-# DATA605 Projects
+## Project 1: Real-time Bitcoin Sentiment Analysis and Predictive Modeling with allms
```

Comment:
Looks good. I would remove it from here and add the text to https://docs.google.com/document/d/1fEd7_oLhFnA5ovzj_HMb9EeMU84nOGEGeWqNRZSz2wo/edit?tab=t.0#heading=h.31ub1xhwl7rv

so we can start editing and improving